src/nft/components/explore/TrendingCollections.tsx

```diff
@@ -0,0 +1,97 @@
+import clsx from 'clsx'
+import { useMemo, useState } from 'react'
+import { useQuery } from 'react-query'
+
+import { Box } from '../../components/Box'
+import { Column, Row } from '../../components/Flex'
+import { headlineMedium } from '../../css/common.css'
+import { fetchTrendingCollections } from '../../queries'
+import { CollectionTableColumn, TimePeriod, VolumeType } from '../../types'
+import CollectionTable from './CollectionTable'
+import * as styles from './Explore.css'
+
+const timeOptions: { label: string; value: TimePeriod }[] = [
+  { label: '24 hour', value: TimePeriod.OneDay },
+  { label: '7 day', value: TimePeriod.SevenDays },
+  { label: '30 day', value: TimePeriod.ThirtyDays },
+  { label: 'All time', value: TimePeriod.AllTime },
+]
+
+const TrendingCollections = () => {
+  const [timePeriod, setTimePeriod] = useState<TimePeriod>(TimePeriod.OneDay)
+
+  const { isSuccess, data } = useQuery(
+    ['trendingCollections', timePeriod],
+    () => {
+      return fetchTrendingCollections({ volumeType: 'eth', timePeriod, size: 100 })
+    },
+    {
+      refetchOnReconnect: false,
+      refetchOnWindowFocus: false,
+      refetchOnMount: false,
+      // refetchInterval: 5000,
```

Comment:
confirming we want the interval commented out?

Reply:
good catch. Was commented out while testing